frontend/public/components/namespace.jsx

```diff
@@ -988,14 +991,16 @@ class NamespaceBarDropdowns_ extends React.Component {
       useProjects,
       children,
       disabled,
+      t,
     } = this.props;
     if (flagPending(canListNS)) {
       return null;
     }
 
     const { loaded, data } = this.props.namespace;
     const model = getModel(useProjects);
-    const allNamespacesTitle = `all ${model.labelPlural.toLowerCase()}`;
+    const allNamespacesTitle =
+      model.label === 'Project' ? t('dropdown~all projects') : t('dropdown~all namespaces');
```

Comment:
We probably want to change the case here

```suggestion
      model.label === 'Project' ? t('dropdown~All Projects') : t('dropdown~All Namespaces');
```